src/ghost/ghost-post.service.ts

```diff
@@ -95,4 +95,19 @@ export class GhostPostService {
             }
         }
     }
+
+    async deleteGhostPost(account: Account, uuid: string) {
+        const apId = account.getApIdForPost({
```

Comment:
`getApIdForPost` is[ marked as deprecated](https://github.com/TryGhost/ActivityPub/blob/main/src/account/account.entity.ts#L40-L43), should we be using an alternative?